src/main/java/pro/sky/course_work2kurs/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package pro.sky.course_work2kurs.service;
+
+import org.springframework.stereotype.Service;
+import pro.sky.course_work2kurs.exception.RequestLimitExceededException;
+import pro.sky.course_work2kurs.exception.RequiredNumberOfQuestionsIsMissingException;
+import pro.sky.course_work2kurs.model.Question;
+
+import java.util.*;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+    private final JavaQuestionService javaQuestionService;
```

Comment:
```suggestion
    private final QuestionService javaQuestionService;
```
по интерфейсу инъекцию надо делать, говорил про это на разборе